docs/developer/protocol/coreliquid.md

```diff
@@ -0,0 +1,314 @@
+| 0x52                     | Bit 3: Combined JCorsair                                |
+| 0x52                     | Bit 4: Combined JPipe1                                  |
+| 0x52                     | Bit 5: Combined JPipe2                                  |
+| 0xB8                     | **Save to device (0 or 1)**                             |
+
+Ligthing effects are sent to the device by sending the feature report `0x52` with the desired data in the above format.
+
+| Byte Value | Lighting Effect Name     |
+|------------|---------------------------|
+| 0          | DISABLE                   |
+| 1          | NO_ANIMATION              |
+| 2          | BREATHING                 |
+| 3          | FLASHING                  |
+| 4          | DOUBLE_FLASHING           |
+| 5          | LIGHTNING                 |
+| 6          | MSI_MARQUEE               |
+| 7          | METEOR                    |
+| 8          | WATER_DROP                |
+| 9          | MSI_RAINBOW               |
+| 10         | POP                       |
+| 11         | JAZZ                      |
+| 12         | PLAY                      |
+| 13         | MOVIE                     |
+| 14         | MARQUEE                   |
+| 15         | COLOR_RING                |
+| 16         | PLANETARY                 |
+| 17         | DOUBLE_METEOR             |
+| 18         | ENERGY                    |
+| 19         | BLINK                     |
+| 20         | CLOCK                     |
+| 21         | COLOR_PULSE               |
+| 22         | COLOR_SHIFT               |
+| 23         | COLOR_WAVE                |
+| 24         | VISOR                     |
+| 25         | RAINBOW                   |
+| 26         | RAINBOW_WAVE              |
+| 27         | VISOR                     |
+| 28         | JRAINBOW                  |
+| 29         | RAINBOW_FLASHING          |
+| 30         | RAINBOW_DOUBLE_FLASHING   |
+| 31         | RANDOM                    |
+| 32         | FAN_CONTROL               |
+| 33         | DISABLE2                  |
+| 34         | COLOR_RING_FLASHING       |
+| 35         | COLOR_RING_DOUBLE_FLASHING|
+| 36         | STACK                     |
+| 37         | CORSAIR_IQUE              |
+| 38         | FIRE                      |
+| 39         | LAVA                      |
+| 40         | END                       |
+
+
+## Fan control
+
+### Fan temperature config - `0x33` (Get) or `0x41` (Set)
+
+Format:
+
+| Byte index | Description |
+| ---------- | ----------- |
+|  0x00 | 0xD0 |
+| 0x01 | 0x32/0x41 (response/write)|
+| 0x02-0x09 | Radiator fan 1 config |
+| 0x0A-0x11 | Radiator fan 2 config |
+| 0x12-0x19 | Radiator fan 3 config |
+| 0x01A-0x21 | Pump speed config |
+| 0x22-0x29 | Waterblock fan config |
+
+A fan temperature config consists of 8 bit integer values:
+  - Mode index
+  - 7 temperature points in Celsius.
+
+### Get fan speed config - `0x32` (Get) or `0x40` (Set)
```

Comment:
I think that naming this section `Fan speed config` in line with the previous one would make more sense, as you can both get and set it.

Reply:
Thank you for spotting this.